src/sunstone/etc/sunstone-views/mixed/user.yaml

```diff
@@ -6,925 +6,966 @@ confirm_vms: false
-            SecurityGroup.edit_labels: true
-            SecurityGroup.menu_labels: true
-    settings-tab:
-        panel_tabs:
-            settings_info_tab: true
-            settings_config_tab: false
-            settings_quotas_tab: true
-            settings_group_quotas_tab: true
-            settings_accounting_tab: true
-            settings_showback_tab: true
-            settings_auth_tab: true
-        actions:
-            # Buttons for settings_info_tab
-            User.update_password: true
-            User.login_token: true
-            User.two_factor_auth: true
-            # Buttons for settings_config_tab
-            Settings.change_language: true
-            Settings.change_password: true
-            Settings.change_view: true
-            Settings.ssh_key: true
-            Settings.login_token: true
-            Settings.two_factor_auth: true
-            # Edit button in settings_quotas_tab
-            User.quotas_dialog: false
-    vmgroup-tab:
-        panel_tabs:
-            vm_group_info_tab: true
-            vm_group_vms_tab: true
-        table_columns:
-            - 0         # Checkbox
-            - 1         # ID
-            - 2         # Name
-            - 3         # Group
-            - 4         # Owner
-            - 5         # Vms
-            #- 6         # Labels
-            #- 7        # Search data
-        actions:
-           VMGroup.create_dialog: true
-           VMGroup.update_dialog: true
-           VMGroup.chmod: true
-           VMGroup.chown: true
-           VMGroup.chgrp: true
-           VMGroup.lockU: true
-           VMGroup.unlock: true
+  dashboard-tab:
+    # The following widgets can be used inside any of the '_per_row' settings
+    # bellow. As the name suggest, the widgets will be scaled to fit one,
+    # two, or three per row. The footer uses the widgets at full size, i.e.
+    # one per row.
+    #
+    # - storage
+    # - users
+    # - network
+    # - hosts
+    # - vms
+    # - groupquotas
+    # - quotas
+    panel_tabs:
+    actions:
+      Dashboard.refresh: false
+      Sunstone.toggle_top: false
+    widgets_one_per_row:
+      - vms
+    widgets_three_per_row:
+    widgets_two_per_row:
+      - network
+      - storage
+    widgets_one_footer:
+  system-top-tab:
+    panel_tabs:
+    actions:
+  users-tab:
+    panel_tabs:
+      user_info_tab: true
+      user_quotas_tab: true
+      user_groups_tab: true
+      user_accounting_tab: true
+      user_showback_tab: true
+      user_auth_tab: true
+    table_columns:
+      - 0 # Checkbox
+      - 1 # ID
+      - 2 # Name
+      - 3 # Group
+      - 4 # Enabled
+      - 5 # Auth driver
+      #- 6         # Password
+      - 7 # VMs
+      - 8 # Memory
+      - 9 # CPU
+      #- 10         # Group ID
+      #- 11        # Hidden User Data
+      #- 12        # Labels
+      #- 13        # Search data
+    actions:
+      User.refresh: true
+      User.create_dialog: true
+      User.update_password: true
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)